From f009c202fa579bf0e0cf9484e11223f249f90f8f Mon Sep 17 00:00:00 2001
From: David Bauer <mail@david-bauer.net>
Date: Thu, 24 Jul 2025 20:25:22 +0200
Subject: [PATCH] hostapd: disable wps button handling

Pressing the WPS button when wpa-supplicant is installed for encrypted
mesh operation triggers WPS in the wpa_supplicant process.

This disables the wireless mesh completely.

Signed-off-by: David Bauer <mail@david-bauer.net>
---
 package/network/config/wifi-scripts/files/etc/rc.button/wps | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/package/network/config/wifi-scripts/files/etc/rc.button/wps b/package/network/config/wifi-scripts/files/etc/rc.button/wps
index 59b1aeff17..fa6e287bee 100755
--- a/package/network/config/wifi-scripts/files/etc/rc.button/wps
+++ b/package/network/config/wifi-scripts/files/etc/rc.button/wps
@@ -1,5 +1,8 @@
 #!/bin/sh
 [ "$ACTION" = "pressed" ] && exit 5
+
+exit 0
+
 for script in /etc/rc.wps/*; do
 	[ -x "$script" ] || continue
 	"$script" && break
-- 
2.51.0

